src/tasks/runstart.ts

```diff
@@ -331,7 +367,20 @@ export const RunStartQuest: Quest = {
     {
       name: "Vote",
       completed: () => have($item`"I Voted!" sticker`) || !get("voteAlways"),
-      do: () => cliExecute("VotingBooth.ash"),
+      do: (): void => {
+        if (myClass() == $class`Pastamancer`) {
+          visitUrl("place.php?whichplace=town_right&action=townright_vote");
+          waitq(1);
+          visitUrl(
+            "choice.php?pwd&option=1&whichchoice=1331&g=" +
+              (random(2) + 1) +
+              "&local[]=2&local[]=2",
+            true,
+            false
+          );
+          cliExecute("set _voteToday = true");
+        } else cliExecute("VotingBooth.ash");
+      },
```

Comment:
I don't like the idea of this. There was a considering way back where I would like it to be possible to run this as a 2-(or more) Day script, and this hardcoding here could be potentially very bad if we picked the bad choices. As mentioned before, we should change this into a structure similar to garbo